DOC-2034: Added TINY-9808 to the release notes. (#2701)
tinymce/tinymce-docs: modules/ROOT/pages/6.5-release-notes.adoc

== Additions
{productname} 6.5 also includes the following additions:

=== New `table_merge_content_on_paste` option which disables the merging behaviour when pasting a table inside an existing table.
//#TINY-9808

In previous versions of {productname}, when a table was copied and pasted inside an existing table, the content of the copied table would fill the existing table instead of being pasted as a whole inside the table cell where the paste was performed. 

> To address this, a new option `table_merge_content_on_paste` has been introduced in {productname} 6.5, that gives users the ability to disable this merging behavior.

Reviewer: I think this would be a bit clearer. To be confirmed by @brian-forte. 

```suggestion
To address this, a new option `table_merge_content_on_paste` has been introduced in {productname} 6.5, that when set to `false` disables this merging behavior.
```

> === New `table_merge_content_on_paste` option which disables the merging behaviour when pasting a table inside an existing table.
> //#TINY-9808
> 
> In previous versions of {productname}, when a table was copied and pasted inside an existing table, the content of the copied table would fill the existing table instead of being pasted as a whole inside the table cell where the paste was performed. 
> 
> To address this, a new option `table_merge_content_on_paste` has been introduced in {productname} 6.5, that when set to `false` disables this merging behavior.
> 
> As a result, table content can now be pasted without merging, ensuring that the entire table is inserted into the desired table cell during the paste operation.

Reviewer 2: ```suggestion
=== New `table_merge_content_on_paste` option which can disable the merging behaviour when pasting a table inside an existing table.
//#TINY-9808

In previous versions of {productname}, when a table was copied and pasted inside an existing table, the content of the copied table would fill the existing table instead of being pasted as a whole inside the table cell where the paste was performed.

{productname} 6.5 includes the new option, `table_merge_content_on_paste`, that allows for this behavior to be turned off.

When this option is set to `false` table merging is disabled and the pasted-in table is inserted into the desired table cell during the paste operation instead.
```
Edits.
